packages/core/src/domain/session/sessionManager.ts

```diff
@@ -32,7 +32,7 @@ export function startSessionManager<TrackingType extends string>(
   computeSessionState: (rawTrackingType?: string) => { trackingType: TrackingType; isTracked: boolean }
 ): SessionManager<TrackingType> {
   tryOldCookiesMigration(options)
-  const sessionStore = startSessionStore(options, productKey, computeSessionState)
+  const sessionStore = startSessionStoreManager(options, productKey, computeSessionState)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could we move the old cookie migration to the cookie store?

Reply:
Yes! Done here: https://github.com/DataDog/browser-sdk/pull/2261/commits/a652aa02c7a79fb465aea70f9be7ff99ab4f3238